lrc/cardano-sl-lrc.cabal

```diff
@@ -83,3 +84,26 @@ library
 
   build-tools: cpphs >= 1.19
   ghc-options: -pgmP cpphs -optP --cpp
+
+test-suite cardano-sl-lrc-test
+  type:                exitcode-stdio-1.0
+  hs-source-dirs:      test
+  main-is:             Spec.hs
```

Comment:
Under the conventions I'm trying to work towards, that file would normally be called "test.hs", but I'm not sure if that works with the `hspec-discover` magic that's being used.

I'm also not sure if this "Spec.hs" file is enough to actually make this a proper test executable. You may been to add a "test.hs" that has a `main` function. Something like `util/test/test.hs`.

You should also make sure that running something like `stack test cardano-sl-lrc` actually does run these tests that you have moved. No point moving them if they aren't going to be run :)